broadcast/internal/arc/arc_policy_quote.go

```diff
@@ -16,8 +17,7 @@ func (a *ArcClient) GetPolicyQuote(ctx context.Context) ([]*broadcast.PolicyQuot
 
 	model, err := getPolicyQuote(ctx, a)
 	if err != nil {
-		a.Logger.Error().Msgf("Failed to get policy quote: %s", err.Error())
-		return nil, err
+		return nil, arc_utils.WithCause(errors.New("GetPolicyQuote: request failed"), err)
```

Comment:
In every other places, you're using `arcutils` (no underscore) - but here it's with-underscored.

Reply:
It's a leftover after author of the file. But you're right I can change it

Reply:
Changed to `arc_utils`